docs/standard/microservices-architecture/multi-container-microservice-net-applications/multi-container-applications-docker-compose.md

```diff
@@ -505,6 +505,8 @@ version: '2'
       command: /bin/bash -c "pushd ./src/Web/WebSPA && npm rebuild node-sass && pushd ./../../.. && dotnet restore ./eShopOnContainers-ServicesAndWebApps.sln && dotnet publish ./eShopOnContainers-ServicesAndWebApps.sln -c Release -o ./obj/Docker/publish"
 ```
 
+* Starting with **.NET Core 2.0**, `dotnet restore` command executes automatically when `dotnet run` or `dotnet build` is executed.
```

Comment:
`dotnet restore` --> the `dotnet restore`
`dotnet run` or `dotnet build` --> `dotnet publish`